src/components/common/Notifications/index.tsx

```diff
@@ -4,17 +4,61 @@ import groupBy from 'lodash/groupBy'
 import { useAppDispatch, useAppSelector } from '@/store'
 import type { Notification } from '@/store/notificationsSlice'
 import { closeNotification, readNotification, selectNotifications } from '@/store/notificationsSlice'
-import type { AlertColor, SnackbarCloseReason } from '@mui/material'
-import { Alert, Link, Snackbar, Typography } from '@mui/material'
+import type { AlertColor, CircularProgressProps, SnackbarCloseReason } from '@mui/material'
+import { Alert, Box, CircularProgress, Link, Snackbar, Typography } from '@mui/material'
 import css from './styles.module.css'
 import NextLink from 'next/link'
 import ChevronRightIcon from '@mui/icons-material/ChevronRight'
 import { OVERVIEW_EVENTS } from '@/services/analytics/events/overview'
 import Track from '../Track'
 import { isRelativeUrl } from '@/utils/url'
+import { type PendingStatus } from '@/store/pendingTxsSlice'
+import useChainId from '@/hooks/useChainId'
+import useSafeAddress from '@/hooks/useSafeAddress'
+import { STATUS_LABELS } from '@/hooks/useTransactionStatus'
 
 const toastStyle = { position: 'static', margin: 1 }
 
+const CircularProgressWithLabel = (props: CircularProgressProps & { value: number }) => {
```

Comment:
I think these 2 components can be removed as they were part of the sticky transactions